src/style/_core.scss

```diff
@@ -1,7 +1,38 @@
 #body {
+
     .btn {
         border-radius: 100px;
         padding-left: 20px;
         padding-right: 20px;
     }
+    .form-group.required .form-label:after { 
```

Comment:
General purpose css classes.  If you add "required" to a form group, then this asterisk will be present.